
Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-sndtj[0m
{"level":"info","ts":1603708791.228008,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1603708791.2290928,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-kq7ch[0m
{"level":"info","ts":1603708791.4958203,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1603708791.4978938,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-25-wmvn4-d2ktc[0m
{"level":"info","ts":1603708797.6312683,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1603708803.2182875,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ cb3b0c80572792ad6a42e4b42e0d09c8fe317b40 in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-25-wmvn4-11[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:cb3b0c80572792ad6a42e4b42e0d09c8fe317b40,2574:3c0867088935cad68d88e023431179bd6e5067bd
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow 3c0867088935cad68d88e023431179bd6e5067bd: cb3b0c80572792ad6a42e4b42e0d09c8fe317b40:
DEBUG: ran git fetch --unshallow origin 3c0867088935cad68d88e023431179bd6e5067bd: cb3b0c80572792ad6a42e4b42e0d09c8fe317b40: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git reset --hard cb3b0c80572792ad6a42e4b42e0d09c8fe317b40
DEBUG: ran git reset --hard cb3b0c80572792ad6a42e4b42e0d09c8fe317b40 in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge 3c0867088935cad68d88e023431179bd6e5067bd
error: error during merge: merging 3c0867088935cad68d88e023431179bd6e5067bd into master: git output: Removing testing/scripts/istio-1.4.2.yaml
Removing testing/s2i/python/transformer/environment_rest
Removing testing/s2i/python/router/environment_rest
Removing testing/s2i/python/model/environment_grpc
Removing testing/s2i/python/combiner/environment_rest
Removing servers/xgboostserver/environment_rest
Removing servers/sklearnserver/samples/iris_grpc.yaml
Removing servers/sklearnserver/environment_rest
Removing servers/mlflowserver/environment_rest
Auto-merging operator/config/manager/configmap.yaml
CONFLICT (content): Merge conflict in operator/config/manager/configmap.yaml
Removing integrations/tfserving/Makefile
Auto-merging helm-charts/seldon-core-operator/values.yaml
CONFLICT (content): Merge conflict in helm-charts/seldon-core-operator/values.yaml
Removing examples/models/mean_classifier/environment_rest
Removing components/routers/epsilon-greedy/environment_rest
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge 3c0867088935cad68d88e023431179bd6e5067bd' command in directory '', output: 'Removing testing/scripts/istio-1.4.2.yaml
Removing testing/s2i/python/transformer/environment_rest
Removing testing/s2i/python/router/environment_rest
Removing testing/s2i/python/model/environment_grpc
Removing testing/s2i/python/combiner/environment_rest
Removing servers/xgboostserver/environment_rest
Removing servers/sklearnserver/samples/iris_grpc.yaml
Removing servers/sklearnserver/environment_rest
Removing servers/mlflowserver/environment_rest
Auto-merging operator/config/manager/configmap.yaml
CONFLICT (content): Merge conflict in operator/config/manager/configmap.yaml
Removing integrations/tfserving/Makefile
Auto-merging helm-charts/seldon-core-operator/values.yaml
CONFLICT (content): Merge conflict in helm-charts/seldon-core-operator/values.yaml
Removing examples/models/mean_classifier/environment_rest
Removing components/routers/epsilon-greedy/environment_rest
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
